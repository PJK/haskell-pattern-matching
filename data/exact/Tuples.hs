module Tuples where

fst :: (a, b) -> a
fst (x, _) = x