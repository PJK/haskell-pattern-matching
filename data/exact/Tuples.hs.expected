{
    "tag": "AnalysisSuccess",
    "contents": [
        [
            "snd",
            {
                "tag": "Redundant",
                "contents": [
                    {
                        "tag": "TuplePattern",
                        "contents": [
                            {
                                "tag": "WildcardPattern",
                                "contents": []
                            },
                            {
                                "tag": "WildcardPattern",
                                "contents": []
                            }
                        ]
                    }
                ]
            }
        ]
    ]
}
